dashboard/src/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>The Dread Pirate Robot's Dashboard++ Sorta</title>
+  <link rel="stylesheet" href="lib/bootstrap-v4.0.0-alpha.6.min.css">
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+  <script type="text/javascript" src="lib/eventemitter2.min.js"></script>
+  <script type="text/javascript" src="lib/roslib.min.js"></script>
+  <script src="lib/vue@2.4.2.min.js"></script>
+  <script type='text/javascript' src="lib/d3.v3.min.js"></script>
+  <script type='text/javascript' src="lib/micropolar-v0.2.2.min.js"></script>
+</head>
+
+<body>
+
+  <audio src="static/Crisp_Ocean_Waves-Mike_Koenig-1486046376.wav" autoplay loop></audio>
+  <audio src="static/flock-of-seagulls_daniel-simion.wav" autoplay loop></audio>
```

Comment:
... :angry: :stuck_out_tongue:

Reply:
Seriously, though, each of these files is over 10MB, and our wifi connection to the boat is often dodgy. Cute idea, but let's get rid of them.

Reply:
I got the same feeling when I pull the repository...